src/components/BrandConfigProvider/BrandConfigProvider.tsx

```diff
@@ -0,0 +1,28 @@
+import React from 'react';
+import { Theme } from './theme/Theme';
+import { ThemeProvider } from './theme/ThemeProvider';
+
+export const BrandConfigContext = React.createContext<{ theme: Theme }>({
+  theme: new Theme(),
+});
+
+interface Props {
+  theme?: Theme;
+  children: React.ReactNode;
+}
+
+export function BrandConfigProvider({ theme, children }: Props) {
+  if (!theme) {
+    theme = new Theme();
+  }
+
+  const context = {
+    theme,
+  };
+
+  return (
+    <BrandConfigContext.Provider value={context}>
+      <ThemeProvider theme={theme}>{children}</ThemeProvider>
+    </BrandConfigContext.Provider>
```

Comment:
I like it!  One can not use the `BrandConfigProvider` and use ThemeProvider (and any other providers encapsulated in here in the future).  But the purpose of BrandConfigProvider is to keep all these things coordinated with each other- I think this is going to work out well.

Reply:
Yes, much better!